index.html

```diff
@@ -1,24 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="wrapper">
-        <h1 class="page-title">Awesome books</h1>
-        <div class="book-list">
-        </div>
-        <form action="./index.html" method="POST" name="book-form">
-          <input type="text" placeholder="Title" name="title" aria-label="fill in book title"><br>
-          <input type="text" placeholder="Author" name="author" aria-label="fill in book author's name"><br>
-          <input type="submit" value="Add" aria-label="Submit the form" class="submitBtn">
-        </form>
+  <div class="wrapper">
+    <nav class="navbar">
```

Comment:
**Optional:**
- [ ] It would be better if you add your app name on the left side of your navigation bar.

 **Wireframe design:**

![a](https://user-images.githubusercontent.com/70483835/165645989-581d3798-62d8-4502-8935-40a097bef937.png)